clearwater-memcached/usr/share/clearwater/clearwater-cluster-manager/plugins/memcached_plugin.py

```diff
@@ -45,8 +45,11 @@
 
 class MemcachedPlugin(SynchroniserPluginBase):
     def __init__(self, params):
-        issue_alarm(alarm_constants.MEMCACHED_NOT_YET_CLUSTERED_MAJOR)
         pdlogs.NOT_YET_CLUSTERED_ALARM.log(cluster_desc=self.cluster_description())
+        self._alarm = alarm_manager.get_alarm(
+            'cluster-manager',
```

Comment:
That's the existing behaviour. I'm not keen to change it as it may affect monitoring software.